### PR TITLE
fix(preview): remove auto-preview experience

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/media/MediaBaseViewer.js [hunk 1]
```diff
@@ -44,7 +44,6 @@ class MediaBaseViewer extends BaseViewer {
         this.toggleMute = this.toggleMute.bind(this);
         this.handleRate = this.handleRate.bind(this);
         this.handleAutoplay = this.handleAutoplay.bind(this);
-        this.mediaendHandler = this.mediaendHandler.bind(this);
```
Comment (ConradJChan) [inline src/lib/viewers/media/MediaBaseViewer.js]: I think leaving in this event is still fine, we just don't need to react to it internally in the preview sdk

Comment (mxiao6) [reply src/lib/viewers/media/MediaBaseViewer.js]: I see

File: src/lib/Preview.js [hunk 2]
```diff
@@ -1239,7 +1239,6 @@ class Preview extends EventEmitter {
                 this.ui.finishProgressBar();
                 break;
             case VIEWER_EVENT.mediaEndAutoplay:
```
Comment (jstoffan) [inline src/lib/Preview.js]: This entire condition can be removed, as it's now a no-op.

Comment (mxiao6) [reply src/lib/Preview.js]: ok

Comment (jstoffan) [reply src/lib/Preview.js]: Actually, it looks like it's preventing the event from being forwarded by the `default` condition.

Comment (mxiao6) [reply src/lib/Preview.js]: Then the test for it should be removed entirely too, right?

Comment (mxiao6) [reply src/lib/Preview.js]: or change it to test the default condition

**Pull request merged**